### PR TITLE
chore: update the condition for removing the label

### PR DESCRIPTION
**Pull request by BoweiZhang-at**

### CODE REVIEW

**Review by BoweiZhang-at — COMMENTED** [review]
This is a comment review from the PR author, which shouldn't remove the label.

File: .github/workflows/state-management2.yml [hunk 1]
```diff
@@ -15,9 +15,9 @@ jobs:
 
   remove-label:
     runs-on: ubuntu-latest
+    # The PR author can leave a comment review, in which case we don't want to remove the label
```
Comment (BoweiZhang-at) [inline .github/workflows/state-management2.yml:18]: Inline comment review from the author - the label shouldn't be removed

File: .github/workflows/state-management2.yml [hunk 2]
```diff
@@ -15,9 +15,9 @@ jobs:
...
     if: |
-      (github.event.pull_request.user.login == 'BoweiZhang-at') &&
-      (contains(fromJSON('["approved", "changes_requested"]'), github.event.review.state)) &&
+      (github.event.pull_request.user.login != github.triggering_actor) &&
```
Comment (psyclaudeZ) [inline .github/workflows/state-management2.yml:20]: But inline comment review from the reviewer will remove the label.

**Review by psyclaudeZ — COMMENTED** [review]
Comment review from the reviewer - the label should be removed

**Review by psyclaudeZ — APPROVED** [review]
Obviously approval from the reviewer should remove the label.

**Review by psyclaudeZ — CHANGES_REQUESTED** [review]
So does request changes from the reviewer - the label should be removed.

**Comment by psyclaudeZ** [comment]
And finally a normal comment from the reviewer shouldn't remove the label.

**Pull request closed without merging**